game/player.py

```diff
@@ -15,17 +17,17 @@ class Player:
 
     def __init__(self):
         """Fills the player's team with a random Pokemon from the PokeAPI"""
-        self.add_to_team(asyncio.run(pokeapi.get_random_pokemon_from_api()))
+        print(self.add_to_team(asyncio.run(pokeapi.get_random_pokemon_from_api())))
         self.lead_pokemon = self.team[0]
         # self.team.extend(asyncio.run(pokeapi.get_random_pokemons_from_api(2)))
 
     def add_to_team(self, pokemon: Pokemon):
-        """Add a pokemon to the user's team and inform the user"""
+        """Add a Pokemon to the user's team and inform the user"""
         self.team.append(pokemon)
-        print(f"{pokemon.name} {MESSAGES['POKEMON']['ADD']}")
+        return f"{pokemon.name} {MESSAGES['POKEMON']['ADD']}"
 
     def remove_from_team(self):
-        """Prompts user to remove a pokemon from the user's team"""
+        """Prompts user to remove a Pokemon from the user's team"""
         is_tossing = True
         while is_tossing:
             if len(self.team) <= 1:
```

Comment:
Change this to `< 1`

Reply:
Changing it to < 1 didn't work. However, changing it `len(self.team) - 1 < 1` did resolve this issue